app/views/admin/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h3 id="links"; style="text-align:left;">&emsp;
+    Admin Dashboard<span style="float:right;"><%= render partial: 'partials/admin_links' %>&emsp;</span>
+</h3>
+
+<style>
+.column {
+  float: left;
+  width: 40%;
+  padding: 50px;
+}
+</style>
```

Comment:
I'd recommend moving this to application.css we also have bootstrap 4 that implements this